sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java

```diff
@@ -287,7 +292,25 @@ public void onCommit() {
 						contact.setContactStatus(ContactStatus.CONVERTED);
 						FacadeProvider.getContactFacade().saveContact(contact);
 					}
-
+					// #1115: pre-fill the case symptoms based on the last visit of the contact (if existent)
+					if(person!=null && disease!=null) {
```

Comment:
Please use the Eclipse code formatter (Ctrl+Shift+F) to properly format your code. We'd like our code style to be consistent.

Reply:
Thanks Mate, Done formatting codes